spec/lib/mvi/service_spec.rb

```diff
@@ -1,9 +1,7 @@
 # frozen_string_literal: true
 
 require 'rails_helper'
-require 'mvi/messages/add_person_message'
```

Comment:
why did these two get removed?

Reply:
The test suite would pass because they were required here. But when MVI::Service was used in runtime these deps were not loaded. So I moved them to the MVI Service and now these deps will be required in dev runtime and testing.

Reply:
That's a good point.  I think I'll take another look at some of the other tests too.  Thanks!